src/components/settings/TagsTab.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <v-container class="wrapper">
+    <div class="title">{{ $tc("caption.custom_tags", 1) }}</div>
+    <p class="mt-2 mb-4">
+      {{ $t("message.add_custom_tags") }}
+    </p>
+    <div class="content">
+      <div class="tag-list">
+        <div class="one" v-for="tag in tags" :key="tag.id">
+          <div class="input-box">
+            <input
+              type="text"
+              placeholder="Add tag"
+              @input="
+                (evt) =>
+                  $store.commit('config/editTagText', {
+                    id: tag.id,
+                    text: evt.target.value,
+                  })
+              "
+              :value="tag.text"
+            />
```

Comment:
Please make this a `vue-tags-input` that they can enter tags directly into.  (Without autocomplete.)